src/main/java/com/google/api/codegen/config/DiscoveryMethodModel.java

```diff
@@ -133,9 +129,13 @@ public String getOutputFullName() {
 
   @Override
   public TypeName getInputTypeName(ImportTypeTable typeTable) {
-    return typeTable
-        .getTypeTable()
-        .getTypeName(((SchemaTypeTable) typeTable).getFullNameFor(method.request()));
+    if (method.request() != null) {
+      return typeTable
+          .getTypeTable()
+          .getTypeName(((SchemaTypeTable) typeTable).getFullNameFor(method.request()));
```

Comment:
Why can't `inputType` be used instead of `method.request()`? (I don't understand the relation between the two of them)

Reply:
Thanks for pointing out the unnecessary complexity here. Just using inputType now.